opal_common/config.py

```diff
@@ -37,6 +37,17 @@ class OpalCommonConfig(Confi):
 
     GIT_SSH_KEY_FILE = confi.str("GIT_SSH_KEY_FILE", str(Path.home() / ".ssh/opal_repo_ssh_key"))
 
+    # Trust self signed certificates (Advanced Usage) -----------------------------
+    # DO NOT change these defaults unless you absolutely know what you are doing!
+    # By default, OPAL only trusts SSL certificates that are signed by a public recognized CA (certificate authority).
+    # However, sometimes (mostly in on-prem setups or in dev environments) users setup their own self-signed certificates.
+    # We allow OPAL to trust these certificates, by changing the following config vars.
+    SELF_SIGNED_CERTIFICATES_ALLOWED=confi.bool(
```

Comment:
Should this be in common ? This would make people think that opal-server also supports self signed